utility/make_rbd_storage_class.py

```diff
@@ -0,0 +1,57 @@
+import os
+from ocs.utils import create_oc_resource
+from utility import templating
+from utility.get_ceph_secret import get_ceph_secret
+from utility.get_ip_addrs import get_ip_addrs_for_ceph_type
+
+
+def make_rbd_pool(cluster_namespace, local_dir):
```

Comment:
Will it be more clear if the function name starts with "create" instead of "make" . Just an observation.